### PR TITLE
staging -> main

### PR DESCRIPTION
**Pull request by samyou-softwire**

### CODE REVIEW

File: HerPortal.ManagementShell/CommandHandler.cs [hunk 1]
```diff
@@ -179,6 +179,42 @@ public void FixAllUserOwnedConsortia()
...
+            outputProvider.Output("The following Consortia will be added to the database:");
+            PrintCodes(consortiumCodesMissingFromDatabase, code => consortiumCodeToConsortiumNameDict[code]);
+        }
+
+        var confirmation = outputProvider.Confirm("Okay to proceed? (Y/N)");
+
+        if (confirmation)
+            adminAction.AddMissingAuthoritiesToDatabase();
+        else
+            outputProvider.Output("No changes made.");
```
Comment (davidm-m) [inline HerPortal.ManagementShell/CommandHandler.cs:215]: `if`s without braces always make me scared. Any chance we could add some in here?

**Review by davidm-m — COMMENTED** [review]
The one thing I've found isn't worth postponing a production release if we've already arranged one, but I would like a PR to fix it to be raised. That way it won't be forgotten about.

File: HerPortal.ManagementShell/CommandHandler.cs (hunk 1 above)
Comment (samyou-softwire) [reply HerPortal.ManagementShell/CommandHandler.cs:215]: linter requirement alas

Comment (samyou-softwire) [reply HerPortal.ManagementShell/CommandHandler.cs:215]: perhaps; the linter rules are not something we've reviewed much before, so can raise a ticket for this. it'd need some work to ensure we're using the same rules for everyone, right now we all agree to use rider's default

Comment (samyou-softwire) [reply HerPortal.ManagementShell/CommandHandler.cs:215]: have done so at https://beisdigital.atlassian.net/browse/PC-1508

Comment (jdgage) [reply HerPortal.ManagementShell/CommandHandler.cs:215]: For what it's worth I agree with this

**Pull request merged**